src/components/RadioItems/RadioItem.css

```diff
@@ -1,19 +1,25 @@
+@import "../../GlobalStyles.css";
```

Comment:
You should be able to just import this file into index.css and by doing that you don't have to import it in all css. This isn't tested so make sure you test it thoroughly.